plugins/net_plugin/net_plugin.cpp

```diff
@@ -1745,7 +1760,10 @@ namespace eosio {
          c->close( false, true );
          return;
       }
-      c->consecutive_rejected_blocks = 0;
+      c->strand.post( [c, blk_num]() {
+         c->last_block_received = blk_num;
```

Comment:
This is set in `process_signed_block` no need to set it here.

Reply:
Removed.